docs/source/tooling/truffle.rst

```diff
@@ -56,21 +56,32 @@ If you install MythX for Truffle in this manner, **you will in addition need to
 
    };
 
-Running
--------
+Usage
+-----
 
 To run MythX for Truffle, run the following command in the root of your configured Truffle project::
 
   truffle run verify
 
 .. note:: The project must compile successfully in order for the plugin to run. If the project hasn't been compiled yet, MythX for Truffle will try to compile it first.
 
-By default, all contracts in the project will be analyzed. To analyze only some of the contracts, append them to the command::
+By default, all contracts in the project will be analyzed. To analyze only some of the contracts, use the following syntax::
 
-  truffle run verify MyContract MyContract2
+  truffle run verify contract.sol
 
-The above command will analyze only the ``MyContract`` and ``MyContract2`` contracts.
+This will analyse all the contracts found in the file ``contract.sol``.
 
+You can also analyze a specific contract::
+
+  truffle run verify contract.sol:MyContract
+
+This will analyse the contract named ``MyContract`` found in the file ``contract.sol``.
+
+.. warning::
+
+   The following syntax has been deprecated and should not be used::
+
+     truffle run verify MyContract
 
 Command options
```

Comment:
While we're updating truffle-security docs, we can probably go through the command options to make sure that they are up to date.

Here is the latest help menu, there have been a few more options added since we last updated this page.

```
Usage: truffle run verify [options] [solidity-file[:contract-name] [solidity-file[:contract-name] ...]]

Runs MythX analyses on given Solidity contracts. If no contracts are
given, all are analyzed.

Options:
  --all      Compile all contracts instead of only the contracts changed since last compile.
  --mode { quick | full }
             Perform quick or in-depth (full) analysis.
  --style { stylish | json | table | tap | unix | markdown | ... },
             Output report in the given es-lint style style.
             See https://eslint.org/docs/user-guide/formatters/ for a full list.
             The markdown format is also included.
  --json | --yaml
             Dump results in unprocessed JSON or YAML format as it comes back from MythX.
             Note: this disables providing any es-lint style reports, and that
             --style=json is processed for eslint, while --json is not.
  --timeout *secs*
             Limit MythX analyses time to *secs* seconds.
             The default is 300 seconds (five minutes).
  --initial-delay *secs*
             Minimum amount of time to wait before attempting a first status poll to MythX.
             The default is 45 seconds.
             See https://github.com/ConsenSys/armlet#improving-polling-response
  --limit *N*
             Have no more than *N* analysis requests pending at a time.
             As results come back, remaining contracts are submitted.
             The default is 4 contracts, the maximum value, but you can
             set this lower.
  --debug    Provide additional debug output. Use --debug=2 for more
             verbose output
             Note: progress is disabled if this is set.
  --min-severity { warning | error }
             Ignore SWCs below the designated level
  --swc-blacklist { 101 | 103,111,115 | ... }
             Ignore a specific SWC or list of SWCs.
  --uuid *UUID*
             Print in YAML results from a prior run having *UUID*
             Note: this is still a bit raw and will be improved.
  --version  Show package and MythX version information.
  --progress, --no-progress
             Enable/disable progress bars during analysis. The default is enabled.
             Note: this is disabled if debug is set.
  --color, --no-color
             Enable/disable output coloring. The default is enabled.
```

Reply:
> While we're updating truffle-security docs, we can probably go through the command options to make sure that they are up to date.

Also a good idea, and let's also separate that out in a separate task. (Trying to keep this PR from getting too big.)